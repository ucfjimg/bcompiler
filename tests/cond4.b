main()
{
    extrn printf;
    auto i;

    i = 0;
    while (i < 3) {
        switch i {
            case 0:
                printf("zero*n");

            case 1:
                printf("one*n");

            case 2:
                printf("two*n");
        }
        i++;
    }
}
