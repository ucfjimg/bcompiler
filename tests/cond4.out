zero
one
two
one
two
two
